### ISSUE TITLE
Node-fetch & webpack

### ISSUE THREAD

**Issue by brianschardt** [Dwolla/dwolla-v2-node#45]
I am having an issue in my project using this package. I believe it is caused by using webpack and node-fetch. The error I am getting is:
`
fetch is not a function

"TypeError: fetch is not a function",
            "    at requestToken (/.../node_modules/dwolla-v2/src/dwolla/Auth.js:37:1)",
            "    at updateToken 
`

You guys implement node fetch like this 
`var fetch = require("node-fetch");`
 According to this: https://github.com/node-fetch/node-fetch/issues/450#issuecomment-387045223
`
// use this
const fetch = require('node-fetch').default
// instead of this
const fetch = require('node-fetch')
`

**Comment by shellscape** [comment]
@sausman please put some eyeballs on this. just ran into this today myself.

**Comment by sausman** [comment]
Hey @shellscape,

Yeah I’ll take a look at this. Any info you can provide on your setup to help reproduce would be great!

Are you using Webpack as well?

**Comment by shellscape** [comment]
@sausman yeah I actually work with @brianschardt but I was on another project with a similar setup. Typically shows up when node-fetch >= 2.0.0 gets the module resolution while bundling. Any version less than 2.0.0 doesn't have the issue. This is caused by a breaking change in 2.0.0 that didn't get much fanfare. See https://github.com/node-fetch/node-fetch/blob/master/docs/CHANGELOG.md#v200 and the last point under General changes.

**Comment by sausman** [comment]
@shellscape Thanks that helps!

One thing I came across you may be able to use is https://www.npmjs.com/package/webpack-node-externals

It was apparently inspired by [this blog post](https://jlongster.com/Backend-Apps-with-Webpack--Part-I) which mentions:

> Webpack will load modules from the node_modules folder and bundle them in. This is fine for frontend code, but backend modules typically aren't prepared for this (i.e. using require in weird ways) or even worse are binary dependencies. We simply don't want to bundle in anything from node_modules.

**Comment by shellscape** [comment]
The one thing we do want to avoid is using externals. We're shipping to Lambda (serverless) and tree shaking the bajeezus out of the bundle to reduce the size of the archive.

**Comment by sausman** [comment]
Hey @shellscape,

We just published `3.1.2` which should fix this!

**Comment by shellscape** [comment]
🙌  and on your day off no less! Very much appreciated!
